rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
Using `named` in rollup config we can have something like:

```js
import glamorous, {ThemeProvider} from 'glamorous'
// like react itself
import React, {Component} from 'react'

```

if you want to have a single default export we will have something like

```js

// in the lib 
Object.assign(
  glamorous,
  {ThemeProvider},
)

// consuming the lib
import glamorous from 'glamorous'
const {ThemeProvider} = glamorous
```

In the end it does not change that much.
@kentcdodds your choice :)